### ISSUE TITLE
Proton-tkg compilation failure if openvr is disabled in cfg

### ISSUE THREAD

**Issue by Kerrung** [Frogging-Family/wine-tkg-git#776]
OS: Arch Linux;
24ff8b8383353feaa2670f715beca076d2977f0c
Proton-tkg compilation failure if openvr is disabled in cfg. And everything works fine if this is switched on. In my opinion, if it is possible to disable openvr in the cfg, this should not cause a crash, or it should not be possible at all. For example, I don't have and don't plan to have any virtual reality hardware, so I would like to disable openvr compilation for myself, but this causes the compilation to fail.
![Screenshot from 2022-05-25 05-04-47](https://user-images.githubusercontent.com/36989404/170152256-4b909353-e247-448e-99a3-e1e68ae192de.png)

**Comment by Tk-Glitch** [comment]
Yeah it used to work but it was made a dependency for the steam helper at some point upstream. It's enforced to enabled for the wine/staging upstream path since that switch, but not for the valve trees. It'll be an easy fix, but indeed fully disabling vr support in proton isn't and won't be supported anymore on the more recent trees.
